### DISCUSSION TITLE
Sophos Filebeat module does not work

### DISCUSSION THREAD

**Discussion by 3isenHeiM** [Security-Onion-Solutions/securityonion#8978] [Unsupported Versions] [answered]
Following the Filebeat configuration module, I've set it accordingly in the `/opt/so/saltstack/local/pillar/minions/<node>.sls` file :

    - module: sophos
      xg:
        enabled: true
        var.input: udp
        var.syslog_host: 0.0.0.0
        var.syslog_port: 9005
        var.default_host_name: firewall.domain.com
        var.known_devices:
          - serial_number: "<REDACTED>"
            hostname: "MY FIREWALL"

However, when doing a `so-filebeat-restart` I get this error :

```
[root@XXX-somgr01 ~]# so-filebeat-restart
=========================================================================
Restarting filebeat...

This could take a while if another Salt job is running. 
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
so-filebeat
so-filebeat
local:
    Data failed to compile:
----------
    Pillar failed to render with the following messages:
----------
    Rendering SLS 'minions.srv03x-somgr01_manager' failed. Please see master log for details.
```

And after this, the container `filebeat` is missing...

What did i do wrong ? It prevents me from having XG logs into SecOn :/

**Comment by dougburks** [comment]
That kind of error usually means that you have a typo somewhere. Please double-check your configuration.

**Reply by 3isenHeiM** [reply]
I copy pasted the section of the file in tjs first post of the discussion. Where could this typo be ? A missing "-" at end last line ?

**Comment by 3isenHeiM** [comment, marked as answer]
Thanks for the hint ! Indeed it was a YAML indentation problem. 

I found it by putting  the file in an online yaml beautifier (https://codebeautify.org/yaml-beautifier).

**TIL yaml only understand SPACES for indentation, not tabs !**
